pkg/controller/perconaservermongodb/ssl.go

```diff
@@ -83,6 +85,36 @@ func (r *ReconcilePerconaServerMongoDB) isCertManagerInstalled(ctx context.Conte
 	return true, nil
 }
 
+func (r *ReconcilePerconaServerMongoDB) isAllStsHasLatestSSL(ctx context.Context, cr *api.PerconaServerMongoDB) (bool, error) {
```

Comment:
Maybe name it like `doAllStsHaveLatestSSL` or maybe `allStsWithLatestSSL`.

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/1383/commits/35c553ed70335e494f9aeb4a55da941c4f542fc6